#   README
### Learning Objectives
  - What are exceptions and when to use them
  - What is the purpose of catching exceptions
  - How to handle exceptions
  - How to use try, catch, finally
  - How to raise built-in exceptions
  - When to implement clean-up actions after exceptions

  ## TASKS
  > 0-safe_list_print/ " prints n elements of a list."
  >
  > 1-divide_print/ "divides 2 integers and prints the result."
  >
  > 2-divide_lists/ "function that divides element by element 2 lists."
  >
  > 3-throw_exception/ "function that throws an Exception."
  >
  > 4-throw_exception_msg/ "throws an Exception with a message."
